breathecode/authenticate/serializers.py

```diff
@@ -1135,6 +1137,16 @@ def validate(self, data: dict[str, str]):
         if syllabus := data.get('syllabus'):
             extra['syllabus'] = syllabus
 
+        have_pending_invites = UserInvite.objects.filter(email=data['email'], status='PENDING').exists()
+        have_accepted_invites = UserInvite.objects.filter(email=data['email'], status='ACCEPTED').exists()
+
+        if not have_accepted_invites and have_pending_invites:
```

Comment:
Separar esta validacion en dos, las accepted si pueden decir `User already exists, go ahead and log in instead.` pero las `PENDING` tiene que decir: `You have a pending invite from {invite.academy.name} that you need to accept before being able to log in. Check your email inbox to accept it or speak to your program manager.`